contents/docs/tutorials/1-minute/survey.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Running Surveys with No Backend
+sidebar: Docs
+showTitle: true
+---
+
+## Objective
+
+Running a user feedback survey without any backend.
+
+## Why is this useful?
+
+Getting user feedback is very important for improving your product. With PostHog, you run a survey without a backend, and can easily pair the survey data with all other metrics about your product.
```

Comment:
With PostHog, you *can* run...

Reply:
I'd probably also add "this is helpful for static sites, such as those written in plain HTML, or generated by a static site generator" or something similar